lib/documents/schemas/veterans_support_organisations.json

```diff
@@ -1,5 +1,5 @@
 {
-  "pre-production": true,
```

Comment:
I'd also remove the pre-production config and logic in a separate commit, once the `target_stack` stuff has been implemented.

EDIT: looks like you've already tried to do this - in which case, just double-check this and rebase it into the (currently) third commit.